### PR TITLE
Reworked LDC notebook so it actually works

### PR DESCRIPTION
**Pull request by hover2pi**
Also added color and name arguments to `LDC.caluclate()` so plots are more customizable.

Have to make sure the Web app didn't break.

### CODE REVIEW

**Comment by hover2pi** [comment]
I also added a new method called `plot_tabs` and removed it from the hard coded Web app so that people can use that plot offline as well.

**Comment by jaymedina** [comment]
nice, I'll take a look at this by Wednesday @hover2pi

**Comment by hover2pi** [comment]
NOTE: The model_grid.p pickles in the EXOCTK_DATA directory need to be deleted when this code is deployed to tlexo and plexo!

**Comment by hover2pi** [comment]
Just had to add some tests. Ok, now it's good to go. Thanks @jaymedina !

File: exoctk/limb_darkening/limb_darkening_fit.py [hunk 1]
```diff
@@ -248,8 +261,8 @@ def calculate(self, Teff, logg, FeH, profile, mu_min=0.05, ld_min=0.01,
         # Make sure the bandpass has coverage
         bp_min = bandpass.wave_min
         bp_max = bandpass.wave_max
-        mg_min = self.model_grid.wave_rng[0]*self.model_grid.wl_units
-        mg_max = self.model_grid.wave_rng[-1]*self.model_grid.wl_units
+        mg_min = self.model_grid.wave_rng[0]
+        mg_max = self.model_grid.wave_rng[-1]
```
[the remark this answers is not in the document]
Comment (hover2pi) [reply exoctk/limb_darkening/limb_darkening_fit.py]: Yeah, the ModelGrip class sorts it.

File: exoctk/tests/test_limb_darkening.py [hunk 2]
```diff
@@ -39,6 +39,24 @@ def test_ldc_object():
...
+
+    # Regular plot
+    fig = ld_session.plot()
+    assert str(type(fig)) == "<class 'bokeh.plotting.figure.Figure'>"
+
+    # Tabbed plot
+    fig = ld_session.plot_tabs()
+    assert str(type(fig)) in ["<class 'bokeh.models.widgets.panels.Tabs'>", "<class 'bokeh.models.layouts.Tabs'>"]
+
+
```
[the remark this answers is not in the document]
Comment (hover2pi) [reply exoctk/tests/test_limb_darkening.py]: Nah, these are just unit tests. I'm just checking the dtype so returning isn't necessary.

**Comment by hover2pi** [comment]
Ok @jaymedina ! I TOTALLY forgot about this. Would you be able to approve it today so I can test it in the Intro-to-ExoCTK notebook? Thanks!

**Comment by hover2pi** [comment]
Nevermind!

**Pull request merged**